Dockerfile

```diff
@@ -1,25 +1,36 @@
 ARG LLVM_VERSION=800
 ARG ARCH=amd64
-ARG BOOTSTRAP=/opt/trailofbits/bootstrap
+ARG UBUNTU_VERSION=18.04
+ARG DISTRO_BASE=ubuntu${UBUNTU_VERSION}
+ARG BUILD_BASE=ubuntu:${UBUNTU_VERSION}
 ARG LIBRARIES=/opt/trailofbits/libraries
-ARG REMILL_INSTALL=/opt/trailofbits/remill
-ARG DISTRO_BASE=ubuntu18.04
 
-#FROM trailofbits/cxx-common/llvm${LLVM_VERSION}-${DISTRO_BASE}-${arch}:latest as base
-FROM trailofbits/cxx-common:llvm${LLVM_VERSION}-${DISTRO_BASE}-${ARCH} as base
-ARG BOOTSTRAP
-ARG LIBRARIES
+# Run-time dependencies go here
+FROM ${BUILD_BASE} as base
+
+RUN apt-get update && \
+    apt-get install -qqy --no-install-recommends libtinfo5 libz3-4 && \
+    rm -rf /var/lib/apt/lists/*
+
+
+# Build-time dependencies go here
+FROM trailofbits/cxx-common:llvm${LLVM_VERSION}-${DISTRO_BASE}-${ARCH} as deps
 ARG LLVM_VERSION
```

Comment:
This looks unused in the actual build steps and can probably be removed, same for the `ARG LIBRARIES` below it

Reply:
Yes, nice catch. Was just doing some sanity checks before I pushed 🙂